profiles/common/modules/custom/nexteuropa_core_views/nexteuropa_core_views.info

```diff
@@ -3,6 +3,5 @@ description = This modules provides views plugins to implement ECL components.
 package = NextEuropa
 core = 7.x
 
-files[] = plugins/card/views_europa_card_plugin_rows.inc
-files[] = plugins/banner/views_europa_banner_plugin_rows.inc
-files[] = plugins/timelines/views_europa_timelines_plugin_rows.inc
+registry_autoload[] = PSR-4
+dependencies[] = drupal:views
```

Comment:
Views is not a submodule of Drupal so this is not valid.

We use this notation when we want to enable a submodule of a module.

Exemple:

`admin_menu:admin_menu_toolbar`